app/src/app/home/home.component.sass

```diff
@@ -14,10 +14,11 @@ a
     text-decoration: underline
 
 div#logos
-  display: flex
+  display: inline-flex
```

Comment:
Making the div inlined does not seem to fix the issue on all screen width. Here is what I get for Samsung S9:
![image](https://user-images.githubusercontent.com/1835665/119099483-3d46db00-ba17-11eb-8fed-ebb265647643.png)
Moreover, it makes the images aligned left on large screens.